docs/guides/runtime/typescript.md

```diff
@@ -29,15 +29,14 @@ Below is the full set of recommended `compilerOptions` for a Bun project. With t
     "verbatimModuleSyntax": true,
     "noEmit": true,
 
-    // Best practices
+    // Linting
     "strict": true,
     "skipLibCheck": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true,
 
     // Some stricter flags
-    "useUnknownInCatchVariables": true,
```

Comment:
still needs `useUnknownInCatchVariables`

Reply:
It is enabled by default with `strict`, so I think using it is meaningless.